src/pycontw2016/settings/base.py

```diff
@@ -318,6 +318,7 @@ def node_bin(name):
     ('NOPREF', _('No preference')),
     ('PREF15', _('Prefer 15min')),
     ('PREF30', _('Prefer 30min')),
+    ('PREF45', _('Prefer 45min')),
```

Comment:
it's better to put a brand new `TALK_PROPOSAL_DURATION_CHOICES` in `settings/production/pycontw-2021.py`.

Reply:
Thanks. I moved it to settings/production/pycontw-2021.py. I also copy the same part to testing/pycontw-2021.py and checked that the result is correct.

Change setting file cmd : 
python /app/src/manage.py runserver 0.0.0.0:8000 --setting=pycontw2016.settings.testing.pycontw2021